hello-world/tests/hello_world.rs

```diff
@@ -0,0 +1,24 @@
+use async_std::prelude::*;
+use async_std::task;
+use std::time::Duration;
+
+#[test]
+fn hello_world() -> anyhow::Result<()> {
+    task::block_on(async {
+        let server =
+            task::spawn(async { hello_world::create_app().listen("localhost:8080").await });
+
+        let client = task::spawn(async {
+            task::sleep(Duration::from_millis(100)).await;
```

Comment:
Seems like a bit of a flakey test? But I guess the app is just for demoing deployment so no big deal

Reply:
Yeah, I still haven't worked out the best way to do this. I was following what they did in their integration tests. Not ideal, and without the delay it does fail every now and then. I haven't seen it fail with the delay. I think what's worse is that parallel tests could clash on the port. You can use port `0` to get an ephemeral port, but no way to know what that port is :-(. These things will improve.